### ISSUE TITLE
Extra quotation marks on output

### ISSUE THREAD

**Issue by Eldar-X** [marko-js/marko#541]
## Extra quotation marks on output of non string objects

When variable is not a string output have extra quotation marks

```
const template = require('template.marko');
template.stream({
test : new Date();
})

// template file

<div data-test="${data.test}"/>
// Output is
<div data-test=""2017-01-19T12:58:42.801Z""></div> 
// In marko v3 output was
<div data-test="2017-01-19T12:58:42.801Z"></div> 
```

**Comment by patrick-steele-idem** [comment]
I pushed a fix. I will be publishing a new version very shortly. Thanks for reporting the problem @Eldar-X.

**Comment by patrick-steele-idem** [comment]
New version published: `marko@4.0.0-rc.6`

Please let us know if you see any other problems. Thanks!
